locales/en/index.yml

```diff
@@ -462,6 +462,7 @@ navigation:
   messages: Messages
   profile: Profile
   accessibility: "Section {{section}}, panel, {{order}} of 4"
+  status: Selected
```

Comment:
It feels more correct `selected: Selected` rather than something as generic as `status`.

Reply:
Agree! I do it right away